cli/cli/kurtosis_package/kurtosis_package_test.go

```diff
@@ -0,0 +1,154 @@
+	require.NoError(t, err)
+
+	kurtosisYamlObj := &enclaves.KurtosisYaml{
+		PackageName:           "",
+		PackageDescription:    "",
+		PackageReplaceOptions: map[string]string{},
+	}
+	err = yaml.UnmarshalStrict(fileBytes, kurtosisYamlObj)
+	require.NoError(t, err)
+	require.Equal(t, packageName, kurtosisYamlObj.PackageName)
+
+	// check main.star file creation
+	expectedMainStarFilepath := path.Join(packageDirpath, "main.star")
+	fileBytes, err = os.ReadFile(expectedMainStarFilepath)
+	require.NoError(t, err)
+	require.Equal(t, mainStarFileContentStr, string(fileBytes))
+}
+
+func TestInitializeKurtosisPackage_IsNotExecutablePackageSuccess(t *testing.T) {
+	packageDirpath, err := os.MkdirTemp("", "package-initialize-test-dir-*")
+	require.NoError(t, err)
+	defer os.RemoveAll(packageDirpath)
+
+	packageName := "github.com/org/my-package"
+	err = InitializeKurtosisPackage(packageDirpath, packageName, isNotExecutablePackage)
+	require.NoError(t, err)
+
+	// check kurtosis.yml file creation
+	expectedKurtosisYamlFilepath := path.Join(packageDirpath, "kurtosis.yml")
+	fileBytes, err := os.ReadFile(expectedKurtosisYamlFilepath)
+	require.NoError(t, err)
+
+	kurtosisYamlObj := &enclaves.KurtosisYaml{
+		PackageName:           "",
+		PackageDescription:    "",
+		PackageReplaceOptions: map[string]string{},
+	}
+	err = yaml.UnmarshalStrict(fileBytes, kurtosisYamlObj)
+	require.NoError(t, err)
+	require.Equal(t, packageName, kurtosisYamlObj.PackageName)
+
+	// check main.star file was not created
+	expectedMainStarFilepath := path.Join(packageDirpath, "main.star")
+	fileBytes, err = os.ReadFile(expectedMainStarFilepath)
+	require.Error(t, err)
+	require.True(t, os.IsNotExist(err))
+	require.Nil(t, fileBytes)
+}
+
+func TestInitializeKurtosisPackage_FailsIfKurtosisYmlAlreadyExist(t *testing.T) {
+	packageDirpath, err := os.MkdirTemp("", "package-initialize-test-dir-*")
+	require.NoError(t, err)
+	defer os.RemoveAll(packageDirpath)
+
+	packageName := "github.com/org/my-package"
+	err = InitializeKurtosisPackage(packageDirpath, packageName, isExecutablePackage)
+	require.NoError(t, err)
+
+	expectedKurtosisYamlFilepath := path.Join(packageDirpath, "kurtosis.yml")
+	fileBytes, err := os.ReadFile(expectedKurtosisYamlFilepath)
+	require.NoError(t, err)
+
+	secondPackageName := "github.com/org/second-package"
+	err = InitializeKurtosisPackage(packageDirpath, secondPackageName, isExecutablePackage)
+	require.Error(t, err)
+	expectedErrorMsgPortion := "'kurtosis.yml' already exist on this path"
+	require.Contains(t, stacktrace.RootCause(err).Error(), expectedErrorMsgPortion)
+
+	kurtosisYamlObj := &enclaves.KurtosisYaml{
+		PackageName:           "",
+		PackageDescription:    "",
+		PackageReplaceOptions: map[string]string{},
+	}
+
+	err = yaml.UnmarshalStrict(fileBytes, kurtosisYamlObj)
+	require.NoError(t, err)
+
+	require.Equal(t, packageName, kurtosisYamlObj.PackageName)
+}
+
+func TestInitializeKurtosisPackage_FailsIfMainStarFileAlreadyExist(t *testing.T) {
```

Comment:
what if a person is trying to make a script a package?

Reply:
based on my read of the code it should work, but can we test that?

Reply:
done